samza-core/src/main/java/org/apache/samza/container/grouper/stream/GroupByPartition.java

```diff
@@ -23,61 +23,50 @@
 import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
-
 import org.apache.samza.config.Config;
 import org.apache.samza.config.TaskConfigJava;
 import org.apache.samza.container.TaskName;
 import org.apache.samza.system.SystemStreamPartition;
 
+/**
+ * An implementation of {@link SystemStreamPartitionGrouper} that groups the input stream partitions according to their partition number.
+ * This leads to a single task processing all the messages of a single partition(e.g. partition 0) across all the input streams.
```

Comment:
"single partition (e.g. partition 0)" <- space before parens open.

Thanks for adding this.

Reply:
Done.